webapp/backend/models.py

```diff
@@ -1,46 +1,108 @@
 from django.db import models
 
+"""
+OBJECTS
+"""
+
 class User(models.Model):
+    """
+    Stores information about every lambda-user.
+    id: the okeanos id of the user.
+    """
     id = models.CharField("UUID", primary_key=True, null=False, blank=False, unique=True, default="", max_length=255, help_text="Unique user id asign by Astakos")
-    token = models.CharField("Okeanos Token", null=True, blank=True, unique=True, default="", max_length=64, help_text="Token provided by ~okeanos.")
+    #token = models.CharField("Okeanos Token", null=True, blank=True, unique=True, default="", max_length=64, help_text="Token provided by ~okeanos.")
 
 class Project(models.Model):
+    """
+    Stores information about every okeanos project that has been used with the LoD.
+    id: the okeanos id of the project.
+    description: a small description of the project.
+    """
     id = models.AutoField("Project ID", primary_key=True, null=False, blank=False, unique=True, default="", help_text="Project id provided by kamaki.")
     description = models.TextField("Project Description", null=True, blank=True, unique=False, default="", help_text="The description of a project.")
 
 class Cluster(models.Model):
+    """
+    Stores every cluster created for the LoD service.
+    id: a unique identifier the service creates for every cluster.
+    :model: models.Server
+    :model: models.Cluster_info
+    """
     id = models.AutoField("Cluster ID", primary_key=True, null=False, help_text="Auto-increment cluster id.")
     # OneToOneField is a ForeignKey with unique=True. Django recommends using OneToOneField instead of a ForeignKey
     # with unique=True.
     master_server = models.OneToOneField('Server', null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
`master_server = models.OneToOneField(Server, null=True, blank=True, on_delete=models.CASCADE)`

Server instead of 'Server'

Same goes for all the foreign key methods